### PR TITLE
Update new backend file for Live Tracking

### PR DESCRIPTION
**Pull request by Khanhvinh1999**
Update new backend file for Live Tracking with api route for frontend 
Could you please help me and approve this PR. Thank you @kcf-jackson

### CODE REVIEW

**Comment by kcf-jackson** [comment]
@Khanhvinh1999 Thanks for the work.

I see that you have two new subfolders "Crowd Monitor_T2_2024", "Live_Tracking_T2". Is the first one obsolete? 
- You should have only one subfolder. 
- Folder name should not have spaces. See README.md at the top level for the naming convention.
- Please remove versioning like T2 and 2024 to encourage continuity of work. 

README.md
- The README.md in "Crowd Monitor_T2_2024" has so many code that it is probably better off to save it as Jupyter notebook. 
    - Edit: I just found out this is actually your update on the documentation, which was not reviewed by me. In general, the documentation is not the place to dump a lot of codes unless you have a specific reason to do so. It's preferred you write a function or a CLI for the users and use that in your documentation. 
    - It's worth mentioning where you get the RTSP URL - a screenshot might be useful here. 
    - I believe it is your teammates in your screenshot. Make sure you have all the permission from everyone to post it.
    - It appears that there is a duplicate of the "Result" section.
    - The "Camo Studio" section can benefit from having a few screenshots on the setup. 
    - For the live camera demo, please provide some information on the regulation related to filming public area.
    - The visualisation is well written. The other sections can follow it. But again, please write a function or a CLI to reduce the many lines of codes you posted in the documentation to a few lines (preferably not more than 5 lines).
    
- The Live_Tracking_T2 folder is missing a README.md file, which should include information on how to start the program and its entry point.

Live_Tracking_T2
- Please add back the missing dependency "python-dotenv".
- Include a screenshot there if you have one.
- Some descriptions / explanations / notebook purpose should be included in the Jupyter Notebook.

The codebase is well-structured, but there is no instruction on how to get things to run. I understand this is Work-In-Progress, so just make sure you add them back later. After you fixed the above, you can tag `ben-AI-cybersec` for merging. Thanks.

File: Crowd_Monitoring/Live_Tracking/Backend_v2/app.py
Comment (Ingabreezy) [inline Crowd_Monitoring/Live_Tracking/Backend_v2/app.py:1]: For production deployments, it is strongly recommended to disable debug mode by setting debug=False. This ensures that sensitive application details are not inadvertently exposed in the event of an error. Given that this code is currently intended for development and testing purposes, running with debug=True is acceptable for now.

**Review by Ingabreezy — APPROVED** [review]
All clear

**Comment by Khanhvinh1999** [comment]
Hi @kcf-jackson, let me update you with the contribution on this PR:

@shricenguyen: develop back end pipeline with IP camera. Contribution to our program development with precise OOP technique. Moreover, he also did code contribution to our website repository. Major role for our success in live stream demo on localhost. Source code optimization and Git version control are so good. 

@Khanhvinh1999: develop data processing technique and data manipulation to convert crowd path into usable information. Troubleshoot and improve program build and integration. Moreover, project management is also applied toward this PR. Doing code review and source code control through Git. 

@MigsMarabut: developed a pipeline to calculate the Expected Goals (xG) metric. It is processed using a pre-trained YOLO model for object detection and keypoint recognition. Homography is applied to convert the image into a 2D plane, enabling the extraction of shot coordinates (x, y) and the angle towards the goal. These inputs are then fed into a logistic regression model, trained on shot angle and distance, to calculate the probability of scoring based on the xG metric.

**Review by Ingabreezy — DISMISSED** [review]

**Comment by kcf-jackson** [comment]
I noticed `Live_Tracking/Live_Tracking/README.md` has a trailing "git" at the end. Other than that, it is a nice revision, and everything looks good to me.

@ben-AI-cybersec Items checked. This is ready for merging. Thanks.

**Comment by kcf-jackson** [comment]
@MigsMarabut has made a PR #48 to fix the issues above, which means this PR is ready to be merged. 

@ben-AI-cybersec This is ready to be merged. Thanks. (Apology I updated the comments several times.)

**Review by ben-AI-cybersec — APPROVED** [review]
lgtm

**Pull request merged**